src/api/catalog.ts

```diff
@@ -58,6 +58,14 @@ export function search({
     default:
       return Promise.reject(new Error(`Unknown data source prefix: '${source}'`))
   }
+  let from = new Date(dateFrom)
```

Comment:
I think this might be a good candidate for moment.js:

```javascript
moment('2010-10-20').isBefore('2010-10-21'); // true
```

Ref: https://momentjs.com/docs/#/query/is-before/